web/src/inputs.ts

```diff
@@ -0,0 +1,19 @@
+import $ from "jquery";
+
+$("body").on("input", ".input-element", function (this: HTMLInputElement, _e: JQuery.Event) {
+    if (this.value.length === 0) {
+        $(this).removeClass("input-element-active");
+    } else {
+        $(this).addClass("input-element-active");
+    }
+});
```

Comment:
Why the name `input-element-active` for "does this input have a current value" -- at least I think that's what you're calculating.

Reply:
The class ".input-element-nonempty" also makes sense to me though, given that's actually what we are checking for before applying the style.

Reply:
I have used JS + CSS to handle this condition since the CSS side of the logic checks if the placeholder is absent as an indication for non empty inputs, but in case we have some input that doesn't have a placeholder in it, the JS will act as the fallback.